etc/apache-mod_proxy_fcgi.conf.template

```diff
@@ -0,0 +1,77 @@
+###
```

Comment:
I have a local branch refactoring `web.py` in order to add CLI unit tests for `web config`. happy to do that as part of that branch and/or open a PR against this one.